docs/gcloud-config.rst

```diff
@@ -0,0 +1,55 @@
+Configuration
+*************
+
+Overview
+========
+
+- Use service :class:`Client <gcloud.client.Client>` objects to configure
+  your applications.
+
+  For example:
+
+  .. code-block:: python
+
+     >>> from gcloud import bigquery
+     >>> client = bigquery.Client()
+
+- :class:`Client <gcloud.client.Client>` objects hold both a ``project``
+  and an authenticated connection to a service.
+
+- The authentication credentials can be implicitly determined from the
+  environment or directly via
+  :meth:`from_service_account_json <gcloud.client.Client.from_service_account_json>`
```

Comment:
Ditto here on using `~` for abbreviation. Since a method, it should've been `from_service_account_json()` anyhow, but with `~` the `:meth:` tag will just handle that for you